backend/src/lib/postcard.js

```diff
@@ -37,13 +37,14 @@ function _sendPostcard(token, recipient, assetStream, message, callbacks) { // e
 /**
  * @param {string} user - Username
  * @param {string} pw - Password
- * @param {function} callback - Call when the auth was successful
+ * @param {function} success - Call when the auth was successful
+ * @param {function} error - Call when the auth failed
  */
 function authorize(user, pw, success, error) {
   SSOHelper.getPostcardcreatorToken(
     user, pw, (err, token) => {
       if (err) {
-        if (typeof error !== undefined) {
+        if (error) {
           error(`Unable to get token: ${err}`)
```

Comment:
I think the code needs to return here. Otherwise it'll call the success callback too, which might lead to that "headers already sent" error!